float64 allowable_cost_per_observation
sensor_msgs/RegionOfInterest roi
geometry_msgs/Pose initial_pose
---
float64 final_cost_per_observation
geometry_msgs/Pose final_pose